decred/tests/unit/dcr/test_account.py

```diff
@@ -185,16 +196,16 @@ def test_tiny_block(prepareLogger):
 
 def newAccount(db):
     # Create an account key
+    testSeed = ByteArray(
+        "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+    ).b
     xk = crypto.ExtendedKey.new(testSeed)
     dcrKey = xk.deriveCoinTypeKey(nets.mainnet)
     acctKey = dcrKey.deriveAccountKey(0)
     acctKeyPub = acctKey.neuter()
     privKeyEncrypted = crypto.encrypt(cryptoKey, acctKey.serialize())
     pubKeyEncrypted = crypto.encrypt(cryptoKey, acctKeyPub.serialize())
-    acct = account.Account(pubKeyEncrypted, privKeyEncrypted, "acctName", "mainnet", db)
-    acct.unlock(cryptoKey)
-    acct.generateGapAddresses()
```

Comment:
I need this function not to unlock the account, moving these lines to the two places where it was called.